grouper/graph.py

```diff
@@ -611,6 +615,13 @@ def get_permission_details(self, name, expose_aliases=True):
                         else:
                             data["service_accounts"][account] = {"permissions": [details]}
 
+            # Add permission audit value
+            permission_audited = False
+            for p in self._permissions.values():
```

Comment:
why are we enumerating `.values()` here? for reference: 
```
        self._permissions = {}  # type: Dict[str, Permission]
```

Reply:
sure! makes sense to iterate over the keys instead and only extract the relevant Permission value.

Reply:
does `self._permissions[name].audited` work? we can just set this inline on L572 then.